src/main/java/pl/allegro/tech/build/axion/release/domain/VersionSorter.java

```diff
@@ -47,10 +57,16 @@ Result pickTaggedCommit(
             for (String tag : tags) {
                 boolean isNextVersion = nextVersionTagPattern.matcher(tag).matches();
                 if (isNextVersion && (ignoreNextVersionTags || ignoreNextVersionOnHead)) {
+                    if (verbose) {
+                        logger.quiet("Ignoring tag: {}, because it's a next version tag and it's not forced", tag);
```

Comment:
I'm not sure which login level would be most appropriate

Reply:
No, the `lifecycle` log level is the default log level [(docs](https://docs.gradle.org/current/userguide/logging.html)). If you log on `quiet`, it will be visible even when running with `./gradlew -q`. It's not a good practice to log everything on `quiet` since users that explicitly want to hide the logs via `-q` switch will see them anyway.

Reply:
my mistake, I forgot about `lifecycle`